tools/cldr-code/src/main/java/org/unicode/cldr/util/FileReaders.java

```diff
@@ -78,6 +78,8 @@ public static String getRelativeFileName(Class<?> class1, String filename) {
             return resourceString.substring(5);
         } else if (resourceString.startsWith("jar:file:")) {
             return resourceString.substring(9);
+        } else if (resourceString.startsWith("wsjar:file:")) {
```

Comment:
@yumaoka did i redo it right? 

(context: ICU4J had the exact same bug and fix…)